package-lock.json

```diff
@@ -1,6483 +1,8 @@
 {
```

Comment:
This file is in `.gitignore`, you should not touch it, unless you have a good reason (?)

Reply:
Done.